plugins/inputs/docker_log/docker_logs.go

```diff
@@ -228,12 +231,13 @@ func (d *DockerLogs) getContainerLogs(
 			}
 			return err
 		}
-		if len(data) > 0 {
+		if num > 0 && len(data) > 0 {
```

Comment:
Just check `num > 0` as it is equal to `len(data)`

Reply:
Taken care of this !